src/bakery.js

```diff
@@ -1,7 +1,13 @@
 export default class Bakery {
 
-  static bake() {
-    document.cookie = 'smartbanner_exited=1';
+  static bake(daysToHide) {
```

Comment:
Ah, ok.
You mean instead calling a seperate static function, we should add the expire date in this way?

`
let expire = Date.now();
expire += hideTtl * 24 * 60 * 60 *1000;
cookie = cookie + '; expires=' + new Date(expire) + ';';
`